### PR TITLE
password hashing

### PR DESCRIPTION
**Pull request by bendesbeus**

### CODE REVIEW

File: sk0-back/src/users/services/password.service.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+@Injectable()
+export class PasswordService {
+  async hashPassword(password: string): Promise<string> {
+    const saltRounds = 10;
+    return bcrypt.hash(password, saltRounds);
+  }
+
+  async comparePasswords(plainTextPassword: string, hashedPassword: string): Promise<boolean> {
+    return bcrypt.compare(plainTextPassword, hashedPassword);
+  }
```
Comment (markkovari) [inline sk0-back/src/users/services/password.service.ts]: I am not convinced that we need to add the suffix "password" to a password service.
It is leading to: `passwordService.hasPassword()`. If this service does nothing else, the password should be in scope and - in my opinion - should not be in the name of the methods. What do you think about this?

**Comment by markkovari** [comment]
The tests need some love in terms of the injected Providers/services. Looks good otherwise. 👼🏿

File: sk0-back/src/users/services/auth.service.spec.ts [hunk 2]
```diff
@@ -84,7 +93,7 @@ describe('AuthService', () => {
 
   describe('createUser', () => {
     it('should return the created user', async () => {
-      const newUser = {
+      const newUser: CreateUserDto = {
```
Comment (markkovari) [inline sk0-back/src/users/services/auth.service.spec.ts:109]: Nice catch 👍🏿

File: sk0-back/src/users/services/password.service.ts (hunk 1 above)
Comment (bendesbeus) [reply sk0-back/src/users/services/password.service.ts]: totally good point, I replace it

**Review by markkovari — APPROVED** [review]

**Review by Jankita58 — APPROVED** [review]

**Review by Fenyo7 — APPROVED** [review]
looks good :D

**Pull request merged**